posts/2017-10-02-serverless-vs-containers.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Serverless (FaaS) vs. Containers (CaaS) - when to pick which?
+description: Docker, Kubernetes, Serverless? Let's discuss the different technologies and discover their up- and downsides. 
+date: 2017-10-02
+layout: Post
+thumbnail: https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/serverless-vs-containers.png
+authors:
+  - PhilippMuns
+---
+
+Contrary to popular thought, FaaS and CaaS have some pretty important things in common.
```

Comment:
Have the full phrase the first time you use it --> `Contrary to popular thought, Functions-as-a-Service (FaaS) and Containers-as-a-Service (CaaS) have some pretty important things in common.`

Also, is CaaS the right framing? I feel like there are few places that really let you run containers straight. Usually you have to set up your whole system like Kubernetes. I'd think of CaaS something more like Azure's Container Service where I can just give you a container and have you run it.